### ISSUE TITLE
Project Templates

### ISSUE THREAD

**Issue by phile314** [ellie-app/ellie#76]
TL;DR: It would be really nice to be able to send other people a link which automatically loads a template into Ellie.

We will teach a short Elm introduction this spring and are currently looking into using Ellie for that. What I have seen so far, it looks really great and well-suited! We usually give the students some smaller tasks with a prepared skeleton for them to complete. The skeleton is the same for everyone, but they should not be able to change the original skeleton. As far as I see, that currently is not possible with Ellie?

I would imagine it would work similar to the current save button, but with an extra option to store it as a template instead.

**Comment by tbash** [comment]
This certainly sounds exciting! What would be the expected outcome of these custom templates? Right now you can save an Ellie, then link to the original. Any saves after that trigger a revision and generate a new link that is different from the first. Could that be what you're looking for?

**Comment by phile314** [comment]
Ah, I indeed tested that but sloppily compared the hashes and only checked the first few characters which did not change. But I see now that the link is actually different, so this already works with Ellie as-is. Thank you for the fast answer :+1:
